README.md

```diff
@@ -315,4 +316,15 @@ The 'acceptance' tests run the full stack, creating, modifying and destroying re
 
 To run all tests, including the acceptance tests, run `make test-acceptance`.
 
-Note that we also run all tests in our [Travis-CI account](https://travis-ci.com/chanzuckerberg/terraform-provider-snowflake).
+We also run all tests in our [Travis-CI account](https://travis-ci.com/chanzuckerberg/terraform-provider-snowflake).
+
+### Pull Request CI
+
+Our CI jobs run the full acceptence test suite, which involves creating and destroying resources in a live snowflake account. Travis-CI is configured with environment variables to authenticate to our test snowflake account. For security reasons, those variables are not available to forks of this repo.
+
+If you are making a PR from a forked repo, you can set up Travis to build it by setting these environement variables:
+
+* `SNOWFLAKE_ACCOUNT` - The account name
+* `SNOWFLAKE_USER` - A snowflake user for running tests.
+* `SNOWFLAKE_PASSWORD` - Password for that user.
+* `SNOWFLAKE_ROLE` - Needs to be ACCOUNTADMIN or similar.
```

Comment:
Maybe add: * 'SNOWFLAKE_REGION` - Defaulted to 'us-west-2', but if different then also add as an environment variable